### PR TITLE
fix: pid display fixes

### PR DESCRIPTION
**Pull request by janrtvld**

### CODE REVIEW

File: apps/easypid/src/app/(app)/activity/index.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { FunkeActivityScreen } from '@easypid/features/activity/FunkeActivityScr
 import { useLocalSearchParams } from 'expo-router'
 
 export default function Screen() {
-  const { host } = useLocalSearchParams()
+  const { did } = useLocalSearchParams()
```
Comment (TimoGlastra) [inline apps/easypid/src/app/(app)/activity/index.tsx:5]: we don't use dids, why did you change this?

**Review by TimoGlastra — CHANGES_REQUESTED** [review]
not sure why the change to did. Maybe entityIdentifier? That could be a did or a https url in that case?

**Comment by janrtvld** [comment]
I was testing with did because for paradym all the hostnames are agent.paradym.id. But this will be fixed anyway with openid fed later.

**Review by TimoGlastra — APPROVED** [review]
